### DISCUSSION TITLE
Linux Auto Lock feature

### DISCUSSION THREAD

**Discussion by gianacta** [ente-io/ente#3929] [Enhancements]
It would be helpful to have an auto-lock feature after a set time duration on the Linux app. This feature exists on the Android app but doesn’t appear to be available on Linux, requiring manual locking.

Linux distro: Debian 12
App: Ente Auth

**Comment by taivlam** [comment]
Could you edit the first post/add a comment to write more details (such as: what type of Linux distro with version number, and is this for the Photos or Auth app)?

(I'm guessing this is Ente Auth, as there is a timed auto-lock on the corresponding Android app and I don't recall that the Auth app locks itself on desktop.)
